perftest/src/test/docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM openjdk:8-jdk-alpine
```

Comment:
do you want to add this changes in this PR? looks like part of your work on perf testing

Reply:
yeah it was a mistake to have this here, but I'm going to delete all the perftest from the main repo, so not worth removing from this PR IMO.